### PR TITLE
*: cleanup content about Special Interest Group

### LINKED ISSUE #521: Cleanup Content about Special Interest Group
subtask of #516

The promotion rules are not well respected where most of contributors think PR numbers requirements are ridiculous. The actual rule is enough sponsor, i.e., consensus voting.

Also the procedure is highly coupled with infra team that you should file an issue first and later two of infra team create and approve the PR so you are added as an member, which hurts SIG committers and leaders notice they responsibility to handle these affairs.

**Proposal**

Remain the consensus voting requirements. SIGs can generate their own guidelines on what they are looking for on committers etc. But hopefully we have loose requirements and contributors conclude how to become a committer by existing examples.

Also let the sig leaders moderate the whole promotion procedure.

### PR DESCRIPTION
**Pull request by tisonkun**
This closes #521

### CODE REVIEW

File: archive/governance/special-interest-groups/sig-web/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Web Special Interest Group(Web SIG)
 
-Web SIG focus on website development around tidb, includding:
+Web SIG focus on website development around tidb, including:
```
Comment (sykp241095) [inline archive/governance/special-interest-groups/sig-web/README.md:3]: Can we delete sig-web all related about sig-web ? (even though they are archived)

File: archive/governance/special-interest-groups/sig-web/roles-and-organization-management.md [hunk 2]
```diff
@@ -2,12 +2,11 @@
 
 ## Join and Promotion
 
-We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
-It lists common requirements, responsibilities and privileges of each role in developer group.
+We mainly follow the instructions in TiDB Developer Group. It lists common requirements, responsibilities and privileges of each role in developer group.
```
Comment (sykp241095) [inline archive/governance/special-interest-groups/sig-web/roles-and-organization-management.md:5]: and the corresponding membership file of sig-web.

File: archive/governance/special-interest-groups/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- [Planner](sig-planner)
+- [SQL-Infra](sig-sql-infra)
+
+### TiDB Product
+
+- [K8S](sig-k8s)
+- [Migrate](sig-migrate)
+- [Diagnosis](sig-diagnosis)
+- [TiUP](sig-tiup)
+- [Web](sig-web)
```
Comment (sykp241095) [inline archive/governance/special-interest-groups/README.md:21]: and this line.

**Review by sykp241095 — CHANGES_REQUESTED** [review]

File: archive/governance/special-interest-groups/sig-web/README.md (hunk 1 above)
Comment (tisonkun) [reply archive/governance/special-interest-groups/sig-web/README.md:3]: Why?

Comment (sykp241095) [reply archive/governance/special-interest-groups/sig-web/README.md:3]: It should be clean up years... ago as repositories in it are not `community oriented`.

Comment (tisonkun) [reply archive/governance/special-interest-groups/sig-web/README.md:3]: Make sense. Updating...

Comment (tisonkun) [reply archive/governance/special-interest-groups/sig-web/README.md]: Updated.

**Review by sykp241095 — APPROVED** [review]

**Review by sykp241095 — APPROVED** [review]
LGTM

**Review by winkyao — APPROVED** [review]
LGTM

**Pull request merged**